assets/src/edit-story/elements/text/edit.js

```diff
@@ -112,21 +113,26 @@ function TextEdit( { content, color, backgroundColor, width, height, x, y, fontF
 	}, [ setPropertiesOnSelectedElements ] );
 
 	// Make sure to allow the user to click in the text box while working on the text.
-	const onClick = ( evt ) => evt.stopPropagation();
+	const onClick = ( evt ) => {
```

Comment:
Comment explaining this code would be useful here.

Reply:
Done